doc/bigint.md

```diff
@@ -0,0 +1,93 @@
+# BigInt
+
+A JavaScript BigInt value.
+
+## Methods
+
+### New
+
+```cpp
+static BigInt New(Napi::Env env, int64_t value);
+static BigInt New(Napi::Env env, uint64_t value);
+```
+
+ - `[in] env`: The environment in which to construct the `BigInt` object.
+ - `[in] value`: The value the JavaScript `BigInt` will contain
+
+These APIs convert the C `int64_t` and `uint64_t` types to the JavaScript
```

Comment:
It would be more consistent with the rest of the doc to start the part that describes the return value with

`Returns ...`

Same goes for the rest of the methods